README.md

```diff
@@ -1,15 +1,74 @@
 # Ionic Langchain
 
-Pre-release. Do not use this.
+Ionic Langchain provides a wrapper around the Ionic Commerce's SDK for use as a `Tool` in a custom Langchain agent.  This tool will enable e-commerce for your agent, allowing your users to ask for product recommendations and purchase products through the agent chat interface.
```

Comment:
@jdmccarty I opted to write my own copy here but please suggest improvements.  As a developer the website copy doesn't tell me a lot in plain english about what the tool is doing.